features/step_definitions/user_steps.rb

```diff
@@ -42,39 +42,44 @@
 Given(/^I am logged into the site$/) do
   visit new_user_session_path
   FactoryGirl.create(:user)
-  fill_in 'Email', with: 'newuser@sb.com' 
-  fill_in 'Password', with: 'password' 
+  fill_in 'Email', with: 'newuser@sb.com'
```

Comment:
At this point, you should probably start using our office's favorite gem [Faker](https://github.com/stympy/faker). It's really easy to get setup and start using. So this would become something like:
```
 fill_in 'Email', with: Faker::Internet.email
```